.travis.yml

```diff
@@ -93,6 +94,9 @@ before_install:
       sed -e '/^coala/d' requirements.txt > requirements.new
       mv requirements.new requirements.txt
     fi
+  # Install astyle linter
+  - sudo apt-get -qq update
+  - sudo apt-get install -y astyle
```

Comment:
I just googled how to install dependencies on travis and I got that. But still didn't work. My builds are failing.